code/modules/html_interface/voting/voting.dm

```diff
@@ -158,6 +167,7 @@ var/global/datum/controller/vote/vote = new()
 	return .
 
 /datum/controller/vote/proc/announce_result()
+	stack_trace("Fuck my shit up. Lock is \[[lock]]")
```

Comment:
Actually that's more lummox' fault but you can blame it on Tom for not creating a sane error handler.